installbuilder/datafiles/Base_DSC.data

```diff
@@ -57,7 +57,7 @@ SHLIB_EXT: 'so'
 /opt/microsoft/omsconfig/Scripts/OMS_MetaConfigHelper.py; intermediate/Scripts/OMS_MetaConfigHelper.py; 755; ${{RUN_AS_USER}}; root
 /opt/microsoft/omsconfig/module_packages/nx_1.0.zip; release/nx_1.0.zip; 755; ${{RUN_AS_USER}}; root
 /opt/microsoft/omsconfig/module_packages/nxOMSPerfCounter_1.1.zip; release/nxOMSPerfCounter_1.1.zip; 755; ${{RUN_AS_USER}}; root
-/opt/microsoft/omsconfig/module_packages/nxOMSSyslog_1.0.zip; release/nxOMSSyslog_1.0.zip; 755; ${{RUN_AS_USER}}; root
+/opt/microsoft/omsconfig/module_packages/nxOMSSyslog_1.1.zip; release/nxOMSSyslog_1.1.zip; 755; ${{RUN_AS_USER}}; root
```

Comment:
@lagalbra 
Actually, I did not change the version number first to give myself enough time to test this with agent E2E.Becuase AMS side version changes I am planning to do next week, the new version will always get overwritten, but since I am done with dogfooding, I bumped up the version too.